library/coverage/result_spec.rb

```diff
@@ -0,0 +1,31 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+config_file = File.expand_path('../fixtures/start_coverage.rb', __FILE__)
```

Comment:
This should use the `fixture` helper, in a `before` block, as in https://github.com/ruby/spec/blob/master/core/module/autoload_spec.rb#L17-L19.